streams/src/main/java/org/apache/kafka/streams/kstream/KTable.java

```diff
@@ -47,16 +48,28 @@
  * interactive queries API.
  * For example:
  * <pre>{@code
- *     final KTable table = ...
+ *     final KTable<String, Long> table = ...;
  *     ...
  *     final KafkaStreams streams = ...;
  *     streams.start()
  *     ...
  *     final String queryableStoreName = table.queryableStoreName(); // returns null if KTable is not queryable
- *     ReadOnlyKeyValueStore view = streams.store(queryableStoreName, QueryableStoreTypes.keyValueStore());
+ *     ReadOnlyKeyValueStore<String, Long> view = streams.store(queryableStoreName, QueryableStoreTypes.keyValueStore());
  *     view.get(key);
```

Comment:
nit: can you remove the blanks? Those would be rendered:
```
    // with blanks code would be intendet
    view.get(key)

// but we don't want indention
view.get(key);
```

This applies to all `<pre>{@code...` sections) -- it was not introduced by you but we should fix it.